frappe/core/doctype/user/user.js

```diff
@@ -76,6 +76,28 @@ frappe.ui.form.on('User', {
 				}
 			}
 		}
+		if (frm.doc.user_emails){
+			var found =0
```

Comment:
Rather than having `Button` in the User, its a much nicer UX to have `frm.add_custom_button`